application/service/services.go

```diff
@@ -70,6 +71,11 @@ type UserService interface {
 	DeprovisionUser(ctx context.Context, username string) (*account.Identity, error)
 }
 
+type NotificationService interface {
```

Comment:
So, as a service we are enforcing that the notification should always be async? ( Ok with this, but just checking )

Reply:
So far we need only async calls. When/if we need sync call then we can add it to the service.

Reply:
So far we need only async calls. When/if we need sync call then we can add it to the service.